app/test_main.py

```diff
@@ -2,0 +1,16 @@
+from unittest import mock
+from typing import Any
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+def test_if_valid_google_url_was_called(mocked_valid_google_url: Any) -> None:
+    can_access_google_page("http://google.com")
+    mocked_valid_google_url.assert_called_once_with("http://google.com")
+
+
+def test_if_has_internet_connection_was_called() -> None:
```

Comment:
The test name `test_if_has_internet_connection_was_called` could be more descriptive. Consider renaming it to something like `test_has_internet_connection_called` to better convey its purpose, as per the checklist suggestion: 'Use descriptive and correct tests names.'